benchmarks/graphics_pipeline/main.cpp

```diff
@@ -913,7 +941,29 @@ void ProjApp::UpdateGUI()
     }
 
     // GUI
-    GetKnobManager().DrawAllKnobs();
+    ImGui::Begin("Knobs");
+    GetKnobManager().DrawAllKnobs(true);
+    DrawExtraInfo();
+    ImGui::End();
 }
 
+void ProjApp::DrawExtraInfo()
+{
+    uint64_t frequency = 0;
+    GetGraphicsQueue()->GetTimestampFrequency(&frequency);
+
+    ImGui::Columns(2);
+    const float gpuWorkDuration = static_cast<float>(mGpuWorkDuration / static_cast<double>(frequency)) * 1000.0f;
+    ImGui::Text("GPU Work Duration");
+    ImGui::NextColumn();
+    ImGui::Text("%f ms ", gpuWorkDuration);
+    ImGui::NextColumn();
+
+    ImGui::Columns(2);
+    const float gpuFPS = static_cast<float>(frequency / static_cast<double>(mGpuWorkDuration));
+    ImGui::Text("GPU FPS");
+    ImGui::NextColumn();
+    ImGui::Text("%f fps ", gpuFPS);
```

Comment:
Just a note: these numbers change every frame, and are not very stable, so it might be hard to read and interpret. 
I am good with this especially as a first iteration or if it serves the needs, but perhaps we could also consider adding average/plot/or other ways to make this easier to understand.